reference/channels.v3.yml

```diff
@@ -2193,7 +2203,7 @@ components:
         example-1: active
     ChannelName:
       type: string
-      description: Name of the channel as it will appear to merchants in the control panel.
+      description: Name of the channel as it will appear to merchants in the control panel. The only special characters it can include are latin alphabet letters, arabic numerals, dashes, and underscores.
```

Comment:
```suggestion
      description: Name of the channel as it will appear to merchants in the control panel. The only special characters it can include are Latin alphabet letters, Arabic numerals, dashes, and underscores.
```